saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/Helpers/MultiModalXmlReader.scala

```diff
@@ -0,0 +1,153 @@
+package edu.illinois.cs.cogcomp.saulexamples.mSpRL2017.Helpers
+
+import edu.illinois.cs.cogcomp.saulexamples.mSpRL2017.Helpers.DataProportion._
+import edu.illinois.cs.cogcomp.saulexamples.mSpRL2017.MultiModalSpRLDataModel.dummyToken
+import edu.illinois.cs.cogcomp.saulexamples.nlp.BaseTypes._
+import edu.illinois.cs.cogcomp.saulexamples.nlp.Xml.NlpXmlReader
+import edu.illinois.cs.cogcomp.saulexamples.nlp.XmlMatchings
+
+import scala.collection.JavaConversions._
+
+/**
+  * Created by taher on 2017-02-28.
+  */
+class MultiModalXmlReader(dataDir: String, proportion: DataProportion) {
+
```

Comment:
I guess this is only based on text, so maybe the prefix multimodel here is redundant?